syntax = "proto3";

package dcmp;

service DecompDispatchService {
    rpc CalculateProblem(AstRequest) returns (CalculationReply) {}
}

message AstRequest {
    string str = 1;
}

message CalculationReply {
}
